scripts/Muon/dummy_label_widget.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import (absolute_import, division, print_function)
+
+
+from Muon import dummy_label_view
+from Muon import dummy_label_presenter
+
+
+class DummyLabelWidget(object):
+
+    """
+    """
+
+    def __init__(self, name, parent=None):
+        view = dummy_label_view.DummyLabelView(name, parent)
+        model = None
+        self.presenter = dummy_label_presenter.DummyLabelPresenter(view, model)
+
+    def getPresneter(self):
```

Comment:
A typo: `getPresneter` -> `getPresenter`.

Can we consider using [property accessors](https://docs.python.org/2/library/functions.html?highlight=property#property) for these types of thing and just having them acessed `.widget` and `.presenter` without the `get`prefix? It's slightly more Pythonic that way.

Reply:
I never knew about this. I will change the code, I agree it looks much neater